src/utils/structuredData.ts

```diff
@@ -0,0 +1,75 @@
+import { AccordionProps } from 'blocks/AccordionBlock/AccordionBlock'
+
+export const structuredFAQPage = (
+  accordions: ReadonlyArray<AccordionProps>,
+) => ({
+  '@context': 'http://schema.org',
+  '@type': 'FAQPage',
+  mainEntity: accordions.map((item) => ({
+    '@type': 'Question',
+    name: item.title,
+    acceptedAnswer: {
+      '@type': 'Answer',
+      text: item.paragraph.html,
```

Comment:
1. "make sure that we don't break this" - TypeScript would warn, no? Unless we forget to also update the types but that would break a lot of things on the site.

    I do think we should look into more of a model-view-controller setup where we avoid passing Storyblok data around in the whole code base.

2. "Is it only used for FAQ" - Interesting. I guess the naming of the block suggests that even if I haven't seen it use in a different way...

    Should we perhaps add a checkbox in Storyblok to indicate that it's an FAQ? Or can it be derived some other way?

Reply:
True, we use Accordions on the jobs page https://www.hedvig.com/se-en/jobs :)

Reply:
I think adding a checkbox for adding schema data could work. Since the presentation, an accordion in this case, is the same for all usage. 👍

Reply:
Have added a field in Storyblok for this!